services/motion/motion.go

```diff
@@ -95,11 +96,6 @@ type ListPlanStatusesReq struct {
 	Extra           map[string]interface{}
 }
 
-// PlanStep represents a single step of the plan
-// Describes the pose each resource described by the plan
-// should move to at that step.
-type PlanStep map[resource.Name]spatialmath.Pose
```

Comment:
@raybjork isn't moving this type from `motion` to `motionplan` technically an SDK-breaking change? Do we need to get approval for this?

Reply:
Bump on this question

Reply:
Yes it is.  We don't have a very strict process for dealing with this kind of breaking change though.

Also, I'm going to basically do this in an upcoming PR anyway